ppl/src/test/java/org/opensearch/sql/ppl/parser/AstBuilderTest.java

```diff
@@ -799,9 +800,9 @@ public void test_batchRCFADCommand() {
   }
 
   @Test
-  public void testShowCatalogsCommand() {
+  public void testShowDataSourcesCommand() {
     assertEqual("show datasources",
-        relation(".CATALOGS"));
+        relation(DATASOURCES_TABLE_NAME));
```

Comment:
why the '.' prefix if not present here ?

Reply:
Moved it to a variable name. 
DATASOURCES_TABLE_NAME  is ".DATASOURCES"